src/ReactiveUI.Tests/ObservableAsPropertyHelperTest.cs

```diff
@@ -229,6 +229,26 @@ public void ToPropertyShouldFireBothChangingAndChanged()
             Assert.Equal("Baz", resultChanged[1].Value);
         }
 
+        private class LazyPropertyFixture : ReactiveObject
+        {
+            public LazyPropertyFixture()
+            {
+                value = Observable
+                    .Return(new Lazy<int>(() => 20 ))
+                    .ToProperty(this, p=>p.Value, new Lazy<int>(()=>25));
+            }
+
+            private readonly ObservableAsPropertyHelper<Lazy<int>> value;
+
+            public int Value => value.Value.Value;
+        }
+
+        [Fact]
+        public void ToPropertyShouldWorkWithLazyProperties()
+        {
+            Assert.Equal( new LazyPropertyFixture().Value, 20 );
```

Comment:
I think this should be `Assert.Equal(expected, actual);` not that it's likely to make much difference...